spec/requests/case_logs_controller_spec.rb

```diff
@@ -792,7 +792,7 @@
         it "downloads logs matching both csv and filter logs" do
           get "/logs?status[]=completed&search=#{postcode}", headers:, params: {}
           csv = CSV.parse(response.body)
-          expect(csv.count).to eq(2)
+          expect(csv.count).to eq(1)
```

Comment:
I don't understand why this was previously `2`.
Only the log matching both status and postcode should be returned.

Reply:
Header row. i.e. 1 log would return 2 rows. 1 row suggests no logs match at all.